CHANGELOG.md

```diff
@@ -29,9 +29,13 @@ Unreleased
 -   Alowing ASCII apostrophes (0x27) in spellings. (\#172).
 -   Only allowing Latin spellings in Maltese lexicon. (\#166).
 -   Split `mon` into Cyrillic and Mongol Bichig (\#179).
--   Added Vietnamese extraction function. (\#181)
--   Modified pron selector in Latin extraction function. (\#183)
-
+-   Added Vietnamese extraction function. (\#181).
+-   Modified pron selector in Latin extraction function. (\#183).
+-   Merged whitelist.py into 'big scrape' script. src scrape.py now checks for
+    existence of whitelist file during scrape to create second 'filtered' tsv.
+    New tsv stored in tsv folder under {"default tsv filename"}_filtered.tsv.
```

Comment:
"TSV" in prose (it's an acronym), thus:

   The new TSV file is placed under `tsv/*_filtered.tsv`.